api/__tests__/routes/round.test.ts

```diff
@@ -0,0 +1,404 @@
+                    await runner.patch({
+                        url: "/round/1",
+                        data: newRound,
+                        expectedResponse: forbiddenResponse,
+                        statusCode: 403,
+                    });
+                });
+
+                test("Cannot reach DELETE /round/:id", async () => {
+                    await runner.delete({
+                        url: "/round/1",
+                        statusCode: 403,
+                    });
+                });
+            });
+
+            describe("Cannot reach any path as a student", () => {
+                beforeEach(() => {
+                    runner.authLevel(AuthenticationLevel.STUDENT);
+                });
+
+                test("Cannot reach GET /round", async () => {
+                    await runner.get({
+                        url: "/round",
+                        expectedData: [forbiddenResponse],
+                        statusCode: 403,
+                    });
+                });
+
+                test("Cannot reach POST /round", async () => {
+                    await runner.post({
+                        url: "/round",
+                        data: newRound,
+                        expectedResponse: forbiddenResponse,
+                        statusCode: 403,
+                    });
+                });
+
+                test("Cannot reach PATCH /round/:id", async () => {
+                    await runner.patch({
+                        url: "/round/1",
+                        data: newRound,
+                        expectedResponse: forbiddenResponse,
+                        statusCode: 403,
+                    });
+                });
+
+                test("Cannot reach DELETE /round/:id", async () => {
+                    await runner.delete({
+                        url: "/round/1",
+                        statusCode: 403,
+                    });
+                });
+            });
+        });
+
+        describe("The requested path must exist", () => {
+            beforeEach(() => {
+                runner.authLevel(AuthenticationLevel.ADMINISTRATOR);
+            });
+
+            test("Find a nonexistent round", async () => {
+                await runner.get({
+                    url: "/round/0",
+                    expectedData: [notFoundResponse],
+                    statusCode: 404,
+                });
+            });
+
+            test("Find a nonexistent round", async () => {
+                await runner.get({
+                    url: "/round/-1",
+                    expectedData: [notFoundResponse],
+                    statusCode: 404,
+                });
+            });
+
+            test("Update a nonexistent round", async () => {
+                await runner.get({
+                    url: "/round/0",
+                    expectedData: [notFoundResponse],
+                    statusCode: 404,
+                });
+            });
```

Comment:
Deze test heet "Update", maar je voert er een GET request uit: dit is dus hetzelfde test als een paar lijntjes boven...